src/Renderer/ThreeExtended/B3dmLoader.js

```diff
@@ -1,5 +1,6 @@
 import * as THREE from 'three';
 import './GLTFLoader';
+import BT from './BatchTable';
```

Comment:
Another proposal:
```js
import './BatchTable';
```
And rename its method to avoid repetition:
```js
BatchTable.parse();
```
(assuming that `BatchTable` is always defined using json format)